src/Events/Telemetry/Telemetry.php

```diff
@@ -195,21 +195,25 @@ public function filter_tec_telemetry_slugs( $slugs ) {
 	 * @return boolean
 	 */
 	public static function is_tec_admin_page(): bool {
+		$current_screen = get_current_screen();
 		$helper = \Tribe__Admin__Helpers::instance();
 
 		// Are we on a tec post-type admin screen?
-		if ( ! $helper->is_post_type_screen( TEC::POSTTYPE ) ) {
+		if (
+			! $helper->is_post_type_screen( TEC::POSTTYPE )
+			//&& ! $helper->is_post_type_screen( TEC::VENUE_POST_TYPE )
+			//&& ! $helper->is_post_type_screen( TEC::ORGANIZER_POST_TYPE )
```

Comment:
```suggestion
			! $helper->is_post_type_screen( TEC::POSTTYPE )
```
Commented out section?

Reply:
Yeah, I wanted to include those but LG decided it wasn't necessary. I should have actually removed them